apps/web/core/components/pages/projects/page-component.tsx

```diff
@@ -266,45 +368,22 @@ function PageComponent() {
 	});
 
 	// Main effect for filtering projects based on archived status
+	// Uses the same filtering logic as ProjectDropDown for consistency
 	useEffect(() => {
 		if (!organizationProjects || organizationProjects.length === 0) {
 			setProjects([]);
 			return;
 		}
 
-		// First, filter out teams/workspaces that are incorrectly returned as projects
-		const actualProjects = organizationProjects.filter((item) => {
```

Comment:
@NdekoCode there were a LOT of logic below, why it's all removed? What was logic vs what it's now?